src/Symfony/Component/DependencyInjection/ParameterBag/ParameterBag.php

```diff
@@ -264,6 +219,9 @@ public function escapeValue($value)
         return $value;
     }
 
+    /**
+     * {@inheritdoc}
+     */
     public function unescapeValue($value)
```

Comment:
The only solution I see to fix this inconsistency is to manage the entire unescaping process in ``resolveValue``